app/src/main/java/com/framgia/forder/screen/shopDetail/ShopDetailFragment.java

```diff
@@ -8,13 +8,26 @@
 import android.view.View;
 import android.view.ViewGroup;
 import com.framgia.forder.R;
+import com.framgia.forder.data.model.Product;
 import com.framgia.forder.data.model.Shop;
+import com.framgia.forder.data.source.ProductRepository;
+import com.framgia.forder.data.source.local.ProductLocalDataSource;
+import com.framgia.forder.data.source.local.realm.RealmApi;
+import com.framgia.forder.data.source.local.sharedprf.SharedPrefsApi;
+import com.framgia.forder.data.source.local.sharedprf.SharedPrefsImpl;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.source.local.sharedprf.SharedPrefsImpl. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports